pallets/liquid-staking/src/lib.rs

```diff
@@ -1813,6 +1852,63 @@ pub mod pallet {
             Ok(inflate_liquid_amount)
         }
 
+        #[require_transactional]
+        fn do_fast_match_unstake(unstaker: &T::AccountId) -> DispatchResult {
+            FastUnstakeRequests::<T>::try_mutate_exists(unstaker, |b| -> DispatchResult {
+                if b.is_none() {
```

Comment:
maybe ok_or is cleaner than `return + take().expect`

Reply:
use `take()` then the value becomes `None`, and it can be deleted if we don't give a new value.